packages/react-core/src/components/Tabs/Tabs.tsx

```diff
@@ -63,11 +83,18 @@ export class Tabs extends React.Component<TabsProps & OUIAProps, TabsState> {
     onSelect: () => undefined as any,
     isFilled: false,
     isSecondary: false,
+    isVertical: false,
+    isBox: false,
     leftScrollAriaLabel: 'Scroll left',
     rightScrollAriaLabel: 'Scroll right',
     variant: TabsVariant.div,
     mountOnEnter: false,
-    unmountOnExit: false
+    unmountOnExit: false,
+    inset: undefined,
+    insetOnMd: undefined,
+    insetOnLg: undefined,
+    insetOnXl: undefined,
+    insetOn2Xl: undefined
```

Comment:
Is there a particular reason these props are defined as `undefined`?

Reply:
Do you recommend setting it to null?    We only want to apply the modifier if one is supplied?

Reply:
Nit: no reason to set defaults to undefined. Just don't specify a default and they will be undefined.